Sources/Transition/HeroTransition+Start.swift

```diff
@@ -46,9 +46,14 @@ extension HeroTransition {
     }
 
     // take a snapshot to hide all the flashing that might happen
-    fullScreenSnapshot = transitionContainer?.window?.snapshotView(afterScreenUpdates: true) ?? fromView?.snapshotView(afterScreenUpdates: true)
-    if let fullScreenSnapshot = fullScreenSnapshot {
-      (transitionContainer?.window ?? transitionContainer)?.addSubview(fullScreenSnapshot)
+    if #available(iOS 11.0, *), inNavigationController {
+      fullScreenSnapshot = transitionContainer?.window?.snapshotView(afterScreenUpdates: false) ?? fromView?.snapshotView(afterScreenUpdates: false)
```

Comment:
Nice. this is much better. I wonder if we should alway use `afterScreenUpdates: false`

Reply:
No, i meant to only use `afterScreenUpdates: false` here. No need to check whether we are `inNavigationController` or iOS 11. The other places needs screen update since we changed some properties and need those properties to reflect in the snapshot. But here we don't.